src/time_series_export/backend/time_series_record.hpp

```diff
@@ -0,0 +1,143 @@
+#ifndef SERENITY_TIME_SERIES_RECORD_HPP_HPP
+#define SERENITY_TIME_SERIES_RECORD_HPP_HPP
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "boost/variant.hpp"
+
+#include "stout/nothing.hpp"
+#include "stout/option.hpp"
+#include "stout/try.hpp"
+
+namespace mesos {
+namespace serenity {
+
+/**
+ * Enum of data series names
+ */
+enum class Series : uint8_t {
+  SLACK_RESOURCES,
+  CPU_USAGE_SYS,
+  CPU_USAGE_USR,
+  CPU_USAGE_SUM,
+  CPU_ALLOC,
+  CYCLES,
+  INSTRUCTIONS,
+  CPI,
+  IPC,
+  CACHE_MISSES,
+  STARTED_TASKS,
+  RUNNING_TASKS,
+  FINISHED_TASKS,
+  REVOKED_TASKS,
+  FAILED_TASKS
+};
+
+
+static std::string SeriesString(Series series) {
+  switch (series) {
```

Comment:
InfluxDB has it's own names... I thought about "Series" like series in https://en.wikipedia.org/wiki/Time_series_database 

Also: I'm thinking about moving nomenclature to more similar like used in OpenTSDB and KairosDB - _Metric_.